docs/template.ejs

```diff
@@ -6,10 +6,6 @@
 
     <meta http-equiv="X-UA-Compatible" content="IE=Edge,chrome=1">
     <meta name="viewport" content="width=device-width, initial-scale=1.0, maximum-scale=1.0, user-scalable=no">
-
-    <meta name="msapplication-TileColor" content="#084d7e">
-    <meta name="msapplication-TileImage" content="/global/images/win8_svcc_logo_white.png">
-    <link rel="shortcut icon" href="images/favicon.ico" type="image/x-icon">
```

Comment:
No need for this anymore?
Does the Webpack Favicon plugin take care of adding the favicon `<link>` or what-not to the final generated HTML for `index.html`?